cdc/cdc/kv/client.go

```diff
@@ -536,7 +536,7 @@ func (s *eventFeedSession) eventFeed(ctx context.Context, ts uint64) error {
 		}
 	})
 
-	tableID, tableName := util.TableIDFromCtx(ctx)
+	// tableID, tableName := util.KeySpanIDFromCtx(ctx)
```

Comment:
ditto

Reply:
There might be something wrong here, I'll check it later.